cmd/rokka/cli/stacks.go

```diff
@@ -270,6 +270,10 @@ func listStacks(c *rokka.Client, args []string) (interface{}, error) {
 	return c.ListStacks(args[0])
 }
 
+func deleteStack(c *rokka.Client, args []string) (interface{}, error) {
+	return "", c.DeleteStack(args[0], args[1])
```

Comment:
Hmm. Do we have a particular reason why we don't return
`nil, c.DeleteStack(args[0], args[1])`?
I've seen this a multiple times so far and maybe we need to correct them (or correct me).

An empty string for me is still content, just empty content. But context wise, nil would be more suitable for this case?

Reply:
you're right. will fix